components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1384,6 +1384,7 @@ struct DecodedSolanaInstructionData {
   uint32 instruction_type;  // Should be converted to either
                             // SolanaSystmeInstruction or SolanaTokenInstruction
                             // based on program ID.
+  array<string> account_params;  // Parameter names of the account meta array.
```

Comment:
Technically it's not part of instruction data, but it's the easiest way to change the current code and pass this to UI. It's related to parsed instruction type in the data anyway, so I don't think it's too bad to put it here.